kotlin_client/src/main/java/com/github/paylike/kotlin_client/dto/HintsResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.github.paylike.kotlin_client.dto
+
+// Describes the hints array received after executing a challenge successfully.
+class HintsResponse(json: Map<String, Any>){
```

Comment:
HintsResponse is
```
{
  "hints": ["first", "second"]
}
```
In which the array is not guaranteed to have multiple members, but guaranteed to have at least one